modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/data_renderers/_comment_renderer.scss

```diff
@@ -0,0 +1,7 @@
+.cell-comment {
+	cursor: pointer;
+
+	svg {
+		pointer-events: none;
+	}
+}
```

Comment:
>Here I ask: why are we not using css modules for this big components?
They use an automatic component scope via Webpack (or any other bundler) and styles are imported in the JS files...
Could be an option for the FDS?

That's an interesting suggestion! IMO, adding a dedicated CSS module makes sense in two cases:
1. If an app has multiple UI (`-web`) modules, that share styles.
2. If CSS can be used independently of app modules. AFAIK, there is no formal 'public API' for CSS, this would be closest to it. But, in this case, we would probably want to place the common styles in `frontend-css` app, or the theme. 

Somewhat grey area is the case when a module has a huge amount of CSS. It may make sense in extreme cases, but normally splitting to components or to subfiles is enough. It's a subjective call of what makes the code more readable. I don't think we need to do this in FDS. 


>I do not like too much the component approach here, as we are not using the CSS in our JS component; this files, alone, do nothing if not imported in the main.scss so it feels more natural to me to be placed along this file.

I highlighted code for this comments thread thinking about just this. I am fairly sure that `cell-comment` class is specific to the `CommentRenderer` component. If this is the case, the best approach is to scope it to the component, so it does not bleed out anywhere else. This includes both FDS and the rest of the page. If we just wrap it in `fds`, like in current PR, it may bleed out to other parts of the FDS. (This component may not be ideal showcase of this principle, as it is very simple. In it, the change boils down to renaming the class to match component name, `.fds-component-renderer`)

Note that the main thing this here is to scope styles, place them within a component-specific container. Where the styles are, in adjecant file or somewhere else, is optional and subjective. A main file with subfiles roughly matching components or parts of UI is not a bad approach. If you think this results in more readable code, let's do it that way then!

There should be very few styles that are not related to any component. Layout styles maybe? Why are we not using Clay layout markup? Theme overrides for independant DXP systems, for example tooltips? Why are we overriding DXP defaults? We should probably prefer to modify FDS to follow them. If there is any such case, please open it for discussion. We may open these are standalone improvement tasks.